content/shmem_finalize.tex

```diff
@@ -15,23 +15,29 @@
 \end{apiarguments}
 
 \apidescription{
-    \FUNC{shmem\_finalize} is a collective operation that ends the \openshmem
-    portion of a program previously initialized by \FUNC{shmem\_init} or \FUNC{shmem\_init\_thread} and
-    releases all resources used by the \openshmem library. This collective
-    operation requires all \acp{PE} to participate in the call. There is an
+    \FUNC{shmem\_finalize} ends the \openshmem
+    portion of a program previously initialized by \FUNC{shmem\_init} or \FUNC{shmem\_init\_thread}.
+   This is a collective
+    operation that requires all \acp{PE} to participate in the call. There is an
     implicit global barrier in \FUNC{shmem\_finalize} to ensure that pending
     communications are completed and that no resources are released until all
     \acp{PE} have entered \FUNC{shmem\_finalize}.
```

Comment:
Do we need to specify which contexts are quiesced? Contexts won't be destroyed until the last call to finalize.